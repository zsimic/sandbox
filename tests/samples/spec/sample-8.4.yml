test:
  strip: |-
   text
  clip: |
    text
      foo
  keep: |+  # some comment
    text
      some over-indented  line

# some comment
      # some indented comment
