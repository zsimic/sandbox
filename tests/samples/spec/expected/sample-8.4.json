{
  "test": {
    "clip": "text\n  foo\n",
    "keep": "text\n  some over-indented  line\n\n",
    "strip": "text"
  }
}